### PR TITLE
Added a note about our BC promise

### PR DESCRIPTION
**Pull request by Nyholm**
We should clearly state what is included and what is not included in our BC promise.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
This should go in the contribution guide. In fact, I think some of this already is?

**Comment by pimjansen** [comment]
Shouldnt we add it in both? Since it will also be relevant for regular use that will just consume only

**Comment by Nyholm** [comment]
Oh. Is it? It really shouldn't be there. :)

It is a promise to all our users. Not just the contributors. 

I'll update the PR

**Comment by bramceulemans** [comment]
The addition of this section looks good to me.

**Review by bramceulemans — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,3 +60,17 @@ for ($i = 0; $i < 3; $i++) {
...
 
 Faker is released under the MIT License. See [`LICENSE`](LICENSE) for details.
+
+## Backward compatibility promise
+
+Faker is using [Semver](https://semver.org/). This means that versions are tagged
+with MAJOR.MINOR.PATCH. Only a new major version will be allowed to break backward
+compatibility (BC).
+
+Classes marked as `@experimental` or `@internal` is not included in the BC promise.
```
Comment (bramceulemans) [inline README.md:70]:
```suggestion
Classes marked as `@experimental` or `@internal` are not included in our backward compatibility promise.
```

**Review by bramceulemans — CHANGES_REQUESTED** [review]

**Comment by pimjansen** [comment]
Lgtm except the comment of bram

**Review by bramceulemans — APPROVED** [review]

**Pull request merged**